### PR TITLE
Proptest

### PR DESCRIPTION
**Pull request by BigKahuna7385**

### CODE REVIEW

File: src/main/java/com/littleBeasts/GameLogic.java [hunk 1]
```diff
@@ -309,6 +319,41 @@ public static AStarPathFinder getCurrentPathFinder() {
...
+        for (IEntity entity : collectionNpc) {
+            if (entity.getName() != null) {
+                if (entity.getName().contains("NPC-")) {
+                    litiInteractables.add(new LitiInteractable(entity, true));
+                } else if (entity.getName().contains("CHEST-")) {
+                    litiInteractables.add(new LitiInteractable(entity, false));
+                }
+            }
+        }
+        int test = 2;
```
Comment (lawand7) [inline src/main/java/com/littleBeasts/GameLogic.java:335]: what is this for? I believe, you can delete it

File: src/main/java/com/littleBeasts/Program2.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    public static IngameScreen getIngameScreen() {
+        return ingameScreen;
+    }
+
+    public static MenuScreen getMenuScreen() {
+        return menuScreen;
+    }
+}
```
Comment (lawand7) [inline src/main/java/com/littleBeasts/Program2.java:72]: why do we need a second programm.java file?

File: src/main/java/com/littleBeasts/entities/LitiPlayer.java [hunk 3]
```diff
@@ -37,6 +38,7 @@
     private int maxHP, currentHP;
     private boolean isFighting;
     private final Image playerPortrait;
+    private ICollisionEntity collisionEntity;
```
Comment (lawand7) [inline src/main/java/com/littleBeasts/entities/LitiPlayer.java:41]: unused attribute i guess

File: src/main/java/com/littleBeasts/Program2.java (hunk 2 above)
Comment (BigKahuna7385) [reply src/main/java/com/littleBeasts/Program2.java:72]: I don't remember... I will investigate.

**Review by lawand7 — APPROVED** [review]
Looks göööd

**Pull request merged**